Docs: Error tracking alerts (#11234)
PostHog/posthog.com: contents/docs/error-tracking/alerts.mdx

---
title: Error tracking alerts
---

To stay on top of issues, you can set up alerts. These enable you to post to Slack, Discord, Teams, or an HTTP Webhook when an issue is created or reopened.

To create an issue alert, go to [error tracking's configuration page](https://us.posthog.com/error_tracking/configuration) and click **Alerting**. This shows you a list of existing alerts. Clicking **New notification** brings you to a page to create a new one.

<ProductScreenshot
    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_04_08_at_11_53_54_2x_81605f7812.png"
    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_04_08_at_11_53_11_2x_8b15ee84e0.png"
    alt="Error tracking alerting"
    classes="rounded"
/>

> Choosing an option brings you to a page to configure the alert. This may require setting up the Slack integration or pasting in a webhook URL. Once done, you can test the alert by clicking **Test function** and then finalize by clicking **Create & enable**.

Reviewer: ```suggestion
Choosing an option brings you to a page to configure the alert. Once done, you can test the alert by clicking **Test function** and then finalize by clicking **Create & enable**.
```

Might I suggest striking this line about "this may require" and instead using a more explanatory screenshot of the actual config?

<img width="1166" alt="Screenshot 2025-04-09 at 1 41 10 PM" src="https://github.com/user-attachments/assets/31479841-e462-4a59-a7fb-b470ff4d99ad" />

Author: I'm just wary of over screenshotting and I don't think if this is more clear without an explanation. "Configure Slack" might be hard to miss.

This will then send alerts to your chosen destination when an issue is created or reopened like this:

![Slack notification](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_04_08_at_14_02_41_2x_2410295066.png)

## Other alerting options

> Since error tracking works by capturing `$exception` events, we can use other features of PostHog to alert you using these events.

Reviewer: > 
```suggestion
Since error tracking works by capturing `$exception` events, any PostHog feature that's triggered by event activity can play a role in alerts.
```

### Real time destinations

The first way is using [real time destinations](/docs/cdp/destinations). This enables you to send events (like `$exception`) to other tools as soon as they are ingested.

> To create a real time destination, go to the [data pipeline tab](https://us.posthog.com/pipeline/overview) in PostHog, click **+ New**, and then select **Destination**. Choose your destination and press **+ Create**.

Reviewer: ```suggestion
To create a real time destination, go to the [data pipelines tab](https://us.posthog.com/pipeline/overview) in PostHog, click **+ New**, and then select **Destination**. Choose your destination and press **+ Create**.
```
